apps/game_backend/lib/game_backend/configuration.ex

```diff
@@ -213,6 +213,24 @@ defmodule GameBackend.Configuration do
     Repo.all(ArenaServer)
   end
 
+  @doc """
+  Returns the list of arena_servers that has the field "status" with a value of "active".
+
+  ## Examples
+
+      iex> list_active_arena_servers()
+      [%ArenaServer{status: "active"}, ...]
+
+  """
+  def list_active_arena_servers do
+    q =
+      from(as in ArenaServer,
+        where: as.status == ^"active"
+      )
+
+    Repo.all(q)
```

Comment:
Do we use this for unity production app? Shouldn't we filter by `environment: "production"`?

Reply:
We don't want that behavior, at least not for the time being. We'll use the `enviroment` field for the ping autodetection only but we want to list the rest of the servers